lib/glific/third_party/gcs/gcs.ex

```diff
@@ -32,30 +32,33 @@ defmodule Glific.GCS do
   @doc """
   Creating a dataset with messages and contacts as tables
   """
-  @spec refresh_gcs_setup(non_neg_integer) :: :ok
+  @spec refresh_gcs_setup(non_neg_integer) :: {:ok, any} | {:error, any}
   def refresh_gcs_setup(organization_id) do
     Logger.info("refresh GCS setup for org_id: #{organization_id}")
 
-    organization_id
-    |> insert_gcs_jobs()
-
-    :ok
+    case insert_gcs_jobs(organization_id) do
+      {:ok, gcs} -> {:ok, gcs}
+      {:error, error} -> {:error, error}
```

Comment:
if we are anyway returning the same tuple can we just use
`insert_gcs_jobs(organization_id)`
instead of case statement from line 39-41